### DISCUSSION TITLE
Suink - Blog Dapp

### DISCUSSION THREAD

**Discussion by jeffierw** [creatorsand/.github#128] [Ideas]
Suink - 基于Walrus的博客平台，类似xLog

一句话 IDEA：基于 Walrus 与 Sui 动态NFT 构建的Dapp，用于发布个人生活和思考。

我的角色：
前端和合约开发

其他角色需求（不包含发起人）： 无

产品是给谁用的？
目标用户为需要发布不可篡改、具有权威性的文章的个人大V、项目方或机构，如区块链项目团队、技术专家（如 Vitalik Buterin）、或希望存档重要内容的企业。

其他产品有什么痛点，用户为什么要选择你的产品？
当前中心化博客或发布平台（如 Medium、个人网站）存在以下痛点：

数据存储中心化：可能因平台政策或技术问题导致文章被删除或丢失。
缺乏权威性：无法保证发布内容未被篡改，难以赢得信任。
访问依赖性：传统博客需依赖 DNS 和服务器，可能受网络封锁或攻击影响。
我们的解决方案：

基于 Walrus 的去中心化存储，保证内容高可用性与不可篡改性。

源码：
- 平台：https://github.com/jeffierw/suink
- 模板：https://github.com/jeffierw/suink-template

**Comment by wufen771** [comment]
根据模版，需要把文档完善好哦，仓库链接贴上去
